client/packages/purchasing/src/purchase_order/DetailView/LineEdit/PurchaseOrderLineEdit.tsx

```diff
@@ -57,21 +56,13 @@ export const PurchaseOrderLineEdit = ({
       paddingBottom={1}
     >
       <Grid size={12} sx={{ mb: 2 }}>
-        {(isUpdateMode && (
-          <BasicTextInput
-            value={`${currentLine?.item?.code}     ${currentLine?.item?.name}`}
-            disabled
-            fullWidth
-          />
```

Comment:
I spent a wee while trying to figure out how to get the full item into this component with the refactor. Then I realised that this was the only thing it's needed for, and this is completely unnecessary as the StockItemSearchInput handles displaying the item just fine (and you only need to pass in the item id for that)